pkg/util/helm_helpers.go

```diff
@@ -127,7 +127,7 @@ func UpdateWithHelm3(releaseName, namespace, chartURL string, vals map[string]in
 		client.Version = ">0.0.0-0"
 	}
 	client.Namespace = namespace
-	client.ReuseValues = true                     // rememeber the values that have been set previously
+	client.ResetValues = true                     // rememeber the values that have been set previously
```

Comment:
wait, don't we need a if clause here to use `ReuseValues` if chart is not upgrade; and use `ResetValues` when upgrading.

Reply:
oh I see, you're always grabbing the values from `helm get values` and always using `resetValues`

can you update that comment to match?

Reply:
okies @yashbhutwala

Reply:
fixed